### PR TITLE
NXEngine: new recipe

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

File: games-arcade/nxengine/nxengine-1.0.0.4.recipe [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/EXL/NXEngine/archive/v1.0.0.4-Rev4.tar.gz"
+SOURCE_URI_2="http://www.cavestory.org/downloads/LIN64-NXEngine-1.0.0.4-Rev-4.tar.gz" # Game Data
+CHECKSUM_SHA256="d467c112e81d4c56337ebf6968bd8bd781bce9140f674e72009a5274d2c15784"
+CHECKSUM_SHA256_2="ddc1f69349bd4239adc6485e0377b503425a7960bbd6696c1b30ea6ed11e1851"
+SOURCE_DIR="NXEngine-$portVersion-Rev4"
+SOURCE_DIR_2="NXEngine"
+PATCHES="nxengine-1.0.0.4.patchset"
+
```
Comment (diversys) [inline games-arcade/nxengine/nxengine-1.0.0.4.recipe]: $portVersion

File: games-arcade/nxengine/nxengine-1.0.0.4.recipe [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+{
+	mkdir -p build_haiku && cd build_haiku
+	qmake ..
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build_haiku
+	mkdir -p $appsDir/nxengine
```
Comment (diversys) [inline games-arcade/nxengine/nxengine-1.0.0.4.recipe]: $appsDir/NXEngine maybe?

File: games-arcade/nxengine/nxengine-1.0.0.4.recipe [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+DESCRIPTION="The platform-adventure game focuses on an amnesiac protagonist \
+who awakens in a cave. Through his explorations, he discovers a plot by the \
+Doctor, a megalomaniac who intends to force the inhabitants of the cave to \
+fight for him in his bid to conquer the world. The protagonist is thrust \
+into the position of savior as he endeavors to defeat the Doctor."
+HOMEPAGE="https://github.com/EXL/NXEngine/"
+COPYRIGHT="2017 Retroguru"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/EXL/NXEngine/archive/v1.0.0.4-Rev4.tar.gz"
```
Comment (diversys) [inline games-arcade/nxengine/nxengine-1.0.0.4.recipe]: $portVersion

**Comment by korli** [comment]
Boo failed: http://vmpkg.haiku-os.org/master/x86_64/buildruns/241/builds/1431.log
which hrev is needed?

**Comment by miqlas** [comment]
@korli 51427 works.

**Pull request merged**